### ISSUE TITLE
NotImplementedError: Cannot convert a symbolic Tensor (up_sampling2d_5_target:0) to a numpy array

### ISSUE THREAD

**Issue by SlowMonk** [tensorflow/tensorflow#37200]
> import numpy as np
> import tensorflow as tf
> print(np.__version__)
> print(keras.__version__)
> print(tf.__version__)
> 
> 1.17.3
> 2.3.1
> 2.1.0

```

import keras.backend as K
from keras.optimizers import Adam
from keras.losses import binary_crossentropy

## intersection over union
def IoU(y_true, y_pred, eps=1e-6):
    #print(y_true)
    if np.max(y_true) == 0.0:
        return IoU(1-y_true, 1-y_pred) ## empty image; calc IoU of zeros

    intersection = K.sum(y_true * y_pred, axis=[1,2,3])
    union = K.sum(y_true, axis=[1,2,3]) + K.sum(y_pred, axis=[1,2,3]) - intersection
    return -K.mean( (intersection + eps) / (union + eps), axis=0)
```

Here the np.max(y_true) bring the problem. 

> NotImplementedError: Cannot convert a symbolic Tensor (up_sampling2d_5_target:0) to a numpy array

**Comment by gadagashwini** [comment]
@SlowMonk, Can you provide the complete standalone code to replicate the issue. Thanks!

**Comment by SlowMonk** [comment]
https://www.kaggle.com/hmendonca/u-net-model-with-submission 
its in17. it work in this kernel but not my local

**Comment by gadagashwini** [comment]
@SlowMonk, Is this similar issue [#37177](https://github.com/tensorflow/tensorflow/issues/37177)?

**Comment by gadagashwini** [comment]
@SlowMonk, Can you provide complete code to analyze this issue and also let us know if its duplicate of [#37177](https://github.com/tensorflow/tensorflow/issues/37177) . Thanks

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by gadagashwini** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by Ceceu** [comment]
@SlowMonk, I had the same issue but it was resolved by updating TensorFlow to 2.2.0rc2 version.
